samples/tfm_integration/psa_firmware/README.rst

```diff
@@ -130,20 +148,33 @@ Sample Output
       [INF] Beginning TF-M provisioning
       [WRN] TFM_DUMMY_PROVISIONING is not suitable for production! This device is NOT SECURE
       [Sec Thread] Secure image initializing!
-      TF-M FP mode: Software
-      Booting TFM v1.5.0
+      Booting TF-M v1.6.0+8cffe127
       Creating an empty ITS flash layout.
       Creating an empty PS flash layout.
-      *** Booting Zephyr OS build v3.0.0-rc1-321-gbe26b6a260d6  ***
+      *** Booting Zephyr OS build zephyr-v3.1.0-3851-g2bef8051b2fc  ***
       PSA Firmware API test
-      Active NS image version: 0.0.0-0
-      Starting FWU; Writing Firmware from 21000000 size 58466 bytes
-      Wrote Firmware; Writing Header from 2100e462 size   432 bytes
+      Active S image version: 0.0.3-0
+      Active NS image version: 0.0.1-0
+      Starting FWU; Writing Firmware from 21000000 size 17802 bytes
+      Wrote Firmware; Writing Header from 2100458a size    16 bytes
       Wrote Header; Installing Image
       Installed New Firmware; Reboot Needed; Rebooting
       [WRN] This device was provisioned with dummy keys. This device is NOT SECURE
       [Sec Thread] Secure image initializing!
-      TF-M FP mode: Software
-      Booting TFM v1.5.0
-      *** Booting Zephyr OS build v3.0.0-rc1-35-g03f2993ef07b  ***
-      Hello World from UserSpace! mps3_an547
+      Booting TF-M v1.6.0+8cffe127
+      *** Booting Zephyr OS build zephyr-v3.1.0-3851-g2bef8051b2fc  ***
+      The version of the PSA Framework API is 257.
+      The minor version is 1.
+      Connect success!
+      TF-M IPC on mps3_an547
+
+Common Problems
+***************
+
+Compilation fails with ``Error: Header padding was not requested...``
+=====================================================================
+
+If you are building this sample, update firmware ``CONFIG_APP_FIRMWARE_UPDATE_IMAGE`` command line
+argument with signed non-secure hex (zephyr_ns_signed.hex), then you will notice the build failure.
+
+Make sure you pass ``unsigned non-secure image (zephyr.hex)`` on ``CONFIG_APP_FIRMWARE_UPDATE_IMAGE``.
```

Comment:
This comment is very unclear to me. It sounds like you get this error when you pass a SIGNED image in, and that we are expecting an UNSIGNED image in this sample? If so, the following is likely clearer:

```suggestion
This error occurs when passing a signed image to ``CONFIG_APP_FIRMWARE_UPDATE_IMAGE``
on the command line, ex: ``zephyr_ns_signed.hex``.

Make sure you pass an unsigned, non-secure image (ex. ``zephyr.hex``) to ``CONFIG_APP_FIRMWARE_UPDATE_IMAGE``.
```